### ISSUE TITLE
"heatmap is not a function" - heatmap.js not loaded with Shiny

### ISSUE THREAD

**Issue by fbreitwieser** [rstudio/d3heatmap#39]
I have a shiny page with a couple of htmlwidgets, including DT and sunburstR, and they work fine. However, I do not manage to get d3heatmap working.

It's strange. When I run the app, I see no errors or warnings in R, but when I look in the browser there's a Javascript error in d3heatmap.js stating that 'heatmap is not a function'. And indeed, heatmap.js is did not get loaded according to my network graph. d3.min.js, datatables.js, and even heatmap.css get loaded. I looked into d3heatmap.yaml and it looks good. I tried it both with the CRAN and github versions.

The file is there:
````
> list.files(system.file("htmlwidgets","lib","d3heatmapcore",package="d3heatmap"))
[1] "heatmap.css" "heatmap.js" 
````

Do you have an idea why the heatmap.js does not get attached?

````
> sessionInfo()
R version 3.2.2 (2015-08-14)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Arch Linux

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] d3heatmap_0.6.1 centrifugeR_0.1 shiny_0.12.2   
````

**Comment by fbreitwieser** [comment]
Update: I tried to just copy the code from heatmap.js into d3heatmap.js, and then it works.

**Comment by Noor59007** [comment]
xlab ylab and title not displaying in d3heatmap()..

**Comment by jcheng5** [comment]
I'm guessing you have an ad blocker installed? It looks like some ad blockers stop /heatmap.js from loading. I don't want to combine the two files but I'll rename it to something else.

**Comment by jcheng5** [comment]
Fixed in 3145f89426a438a1b3831053223d1fecf9e18a1c
